dbms/src/Common/TiFlashMetrics.h

```diff
@@ -212,6 +212,11 @@ static_assert(RAFT_REGION_BIG_WRITE_THRES * 4 < RAFT_REGION_BIG_WRITE_MAX, "Inva
       "The freshness of tiflash data with tikv data",                                                                               \
       Histogram,                                                                                                                    \
       F(type_syncing_data_freshness, {{"type", "data_freshness"}}, ExpBuckets{0.001, 2, 20}))                                       \
+    M(tiflash_memory_usage_by_class,                                                                                                \
+      "TiFlash memory consumes by class",                                                                                           \
+      Gauge,                                                                                                                        \
+      F(type_uni_page_ids, {"type", "uni_page_ids"}),                                                                               \
+      F(type_versioned_entry_or_delete, {"type", "versioned_entry_or_delete"}))                                                     \
```

Comment:
```suggestion
      F(type_versioned_entries, {"type", "versioned_entries"}))                                                     \
```
I'd like to shorten the name here for representing in grafana